src/brush.rs

```diff
@@ -41,6 +41,17 @@ pub enum BrushRef<'a> {
     Image(&'a Image),
 }
 
+impl<'a> BrushRef<'a> {
+    /// Converts the reference to an owned brush.
+    pub fn to_owned(&self) -> Brush {
```

Comment:
Should this be [`ToOwned::to_owned`](https://doc.rust-lang.org/std/borrow/trait.ToOwned.html)?

Reply:
I wish it could, but `Brush` would need to implement `Borrow` which requires the borrowed type to be a reference and we can't satisfy that here since `BrushRef` is a distinct type.